plugin/evm/config/config.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/coreth/params"
```

Comment:
I believe this will cause problems with libevm. Let's try to work around it instead

Reply:
right :/